.travis.yml

```diff
@@ -4,9 +4,6 @@
 language: objective-c
 
 env:
-  env:
-  matrix:
-    - CONDA_PY=27
```

Comment:
The reason this is removed relates to the fact that m2crypto is being skipped on Linux and OSX (``skip: True  # [not win or py3k]``). Given our conversation in #4, is this the wrong way around?